### DISCUSSION TITLE
extra /run/user/... directory created

### DISCUSSION THREAD

**Discussion by timsoftgit** [crosswire/xiphos#1123] [Q&A]
hi, I noticed when building xiphos for a distribution package that during the make part of the build (after cmake but before make install) that a /run/user/0/dconf/ directory gets created and in it a user file containing a couple of bytes.
does someone know why is this getting created and how I can stop it.?
(yes I'm aware its on a tmpfs, but creating out of tree files and not tidying up is something I would like to avoid).
many thanks, Tim

**Comment by karlkleinpaste** [comment]
I'm completely unfamiliar with this. My systems don't have /var/user in the first place, though they have /run/user.

Anyone else recognize this?

**Comment by timsoftgit** [comment]
my appologies, my title was correct, but in the details it should also read /run/user/...

**Comment by karlkleinpaste** [comment]
In that case, /run/user/0 is per-user files for the root (id 0) user. I've not seen anything get created during a build process in /run/user (that I recall) but perhaps someone else has other knowledge or memories. @greg-hellings?

**Comment by timsoftgit** [comment]
i only noticed it since I started testing builds in a overlay chroot, which enables me to see everything written outside expected locations. i run cmake with -DDBUS=OFF option, and i've also tried without that option for testing, but I still get the "user" file written in /run/user/0/dconf/
it looks like it might be related to elogind and dbus calling dconf but why would make do this, and where.? unfortunately greping the source for dconf doesn't help.
